src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/If.cs

```diff
@@ -60,9 +60,8 @@ public override bool IsLazyEvalParam(int index)
             return index >= 1;
         }
 
-        public override bool CheckInvocation(TexlBinding binding, TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
+        public override bool CheckTypes(BindingConfig config, TexlNode[] args, DType[] argTypes, IErrorContainer errors, out DType returnType, out Dictionary<TexlNode, DType> nodeToCoercedTypeMap)
         {
-            Contracts.AssertValue(binding);
```

Comment:
>binding [](http://example.com/codeflow?start=34&length=7)

AssertValue(config). Same comment in other places

Reply:
Yes, this has been fixed